### ISSUE TITLE
BitMax

### ISSUE THREAD

**Issue by ctosae** [krijnsent/crypto_vba#68]
Hi, your project is wonderful!
I'm using it with Kraken and Exchange.
Is it possible to implement BitMax as exchange?
https://bitmax-exchange.github.io/bitmax-pro-api/#bitmax-pro-api-documentation
Regards

**Comment by krijnsent** [comment]
Will have a look at it, it seems to be quite a sizeable exchange. I have no forecast on when I can put it on my agenda, as my priorites are at the moment with my job.

**Comment by ctosae** [comment]
Yes, for example it's the primary exchange for Akash (The Unstoppable Cloud) Cosmos/IBC blockchain based.
Thank you for considering the this implementation

**Comment by ctosae** [comment]
Update: the new brand is "AscendEX" (ex BitMax)
